docs/development/README.adoc

```diff
@@ -54,6 +54,127 @@ then run this script 3 times with a different `<address>`.
 
 TBD
 
+==== Configuration
+
+===== Application
+
+The client expects parameters to be configured in a config file or passed as environment
+variables.
+
+Application configuration can be stored in a file and passed to the application
+run command with the `--config` flag. Example:
+[source,bash]
+----
+./keep-client --config /path/to/your/config.toml start
+----
+
+====== Operator's Ethereum Key File
+
+Ethereum Key File is expected to be encrypted with a password. The password has
+to be provided in prompt after the client starts or configured as environment
+variable `KEEP_ETHEREUM_PASSWORD`.
+
+====== Parameters
```

Comment:
Thinking more about it, I would focus on making sure the client binary `--help` has sufficient information and then just copy it here, just like we refer to the sample config file. [Go-ethereum is doing the same](https://geth.ethereum.org/docs/interface/command-line-options) and I think it's a good practice: ensures that the client CLI help is in a good shape, and we are not reinventing the wheel.